announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,41 @@
+
+Cluster API is a Kubernetes sub-project focused on providing declarative APIs and tooling to simplify provisioning, upgrading, and operating multiple Kubernetes clusters.
+
+Started by the Kubernetes Special Interest Group (SIG) Cluster Lifecycle, the Cluster API project uses Kubernetes-style APIs and patterns to automate cluster lifecycle management for platform operators. The supporting infrastructure, like virtual machines, networks, load balancers, and VPCs, as well as the Kubernetes cluster configuration are all defined in the same way that application developers operate deploying and managing their workloads. This enables consistent and repeatable cluster deployments across a wide variety of infrastructure environments. The project was started over 2 years ago, and is growing and improving rapidly. You can find more details [here](https://cluster-api.sigs.k8s.io/).
+
+### Why is Giant Swarm moving to Cluster API?
+
+In 2017, due to a lack of better alternatives, Giant Swarm decided to build their product based on their own implementation of automation, tools, and operators. This allowed Giant Swarm to provide a stable and easy-to-use product that you are currently using to manage your Kubernetes clusters. Although this solution was ideal for a long time it has its downsides:
+- You have limited control over configuration of individual parts of the infrastructure. The product is not flexible enough and sometimes doesn't support all of your use cases.
+- Adding new functionality is time-consuming and as hyperscalers (Amazon, Microsoft) expand their offering it becomes increasingly difficult to keep up and make it all available to you in a timely manner.
+- Your cluster management is limited to what Giant Swarm tooling enables and there are no available alternatives.
+- Although Giant Swarm provides pure "vanilla" Kubernetes clusters, it is currently not possible to move the workload clusters and have them managed by another product without migration. 
+
+Giant Swarm has been closely watching the development of the upstream Cluster API project and since it achieved a satisfying level of maturity, decided to adopt it in order to address all of the downsides mentioned above and bring more value to you:
+- Significantly more flexibility and configurability over your infrastructure. Things like deploying into existing networks or customizing api-server configuration directly will now be possible.
+- A future-proof API that protects you from vendor lock-ins, developed by a dynamically growing community which will increase the speed of adding new functionality over time.
+- The possibility to use all tooling compatible with the Cluster API -- not only that created by Giant Swarm.
+- Ability to create and manage workload clusters with managed control planes like EKS or AKS.
+- Adding new providers will become easier and Giant Swarm will be in a better position to expand their offering beyond AWS, Azure, and current On-Premises solutions.
```

Comment:
```suggestion
- Adding new providers will become easier and Giant Swarm will be in a better position to expand their offering beyond AWS, Azure, and current On-Premises solutions.
```
Does `their` refer to Giant Swarm or other provider offerings? If the first needs to change to `our`

Reply:
```suggestion
- Adding new providers will become easier and Giant Swarm will be in a better position to expand our offering beyond AWS, Azure, and current On-Premises solutions.
```

Reply:
there's no "our" or "us" anywhere else in the document, are you sure this fits here? It's written in 3rd person so their = Giant Swarm's